Panels/tests/Header-specs.js

```diff
@@ -69,15 +54,15 @@ describe('Header Specs', () => {
 			</Header>
 		);
 
-		const actual = subject.find(`.${css.slotAbove}`).first().text();
+		const slotAbove = screen.getByText(expected);
 
-		expect(actual).toBe(expected);
+		expect(slotAbove).toHaveClass(css.slotAbove);
```

Comment:
Maybe create `expected` variables like above (for css classes) for consistency. For the others below as well.